skimage/metrics/_structural_similarity.py

```diff
@@ -165,9 +165,14 @@ def structural_similarity(im1, im2,
 
     if np.any((np.asarray(im1.shape) - win_size) < 0):
         raise ValueError(
-            "win_size exceeds image extent.  If the input is a multichannel "
-            "(color) image, set channel_axis to the axis number corresponding "
-            "to the channels.")
+            "win_size exceeds image extent.",
+            "Either ensure that your images are ",
+            "at least 7x7; or pass win_size explicitly",
+            " in the function call, with an odd value ",
```

Comment:
```suggestion
            "at least 7x7; or pass win_size explicitly ",
            "in the function call, with an odd value ",
```
just out of consistency ;)